packages/debug/src/common/debug-configuration.ts

```diff
@@ -72,6 +72,21 @@ export interface DebugConfiguration {
 
     /** Task to run after debug session ends */
     postDebugTask?: string | TaskIdentifier;
+
+    /**
+     * When true, a save will not be triggered for open editors when starting a debug session,
+     * regardless of the value of the `debug.saveBeforeStart` setting.
+     */
+    suppressSaveBeforeStart?: boolean;
+
+    /** When true, the debug toolbar will not be shown for this session. */
```

Comment:
As mentioned above, I don't think this flag makes sense in Theia. If Theia does not use it, It shouldn't appear in Theia code.